recipes/ldas-tools-cmake/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ldas-tools-cmake" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "304d844b7ef165ccb1b1e90ff7ee7110bd53e17e64b3b392a530c1e89a2f80ed" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake >=3.6
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - exit $(test -f ${PREFIX}/share/ldas-tools/cmake/Modules/LDASToolsInstallDirs.cmake)  # [not win]
+    - exit $(test -f ${PREFIX}/share/ldas-tools/cmake/Modules/Autotools/ArchiveX/cx_ldas_pkg_check.cmake)  # [not win]
+    - if not exist %LIBRARY_PREFIX%\share\ldas-tools\cmake\Modules\LDASToolsInstallDirs.cmake exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\share\ldas-tools\cmake\Modules\Autotools\ArchiveX\cx_ldas_pkg_check.cmake exit 1  # [win]
+
+about:
+  home: https://git.ligo.org/ldastools/LDAS_Tools
+  license: Other
```

Comment:
Please add the ``license_file`` here

Reply:
@marcelotrevisani, the distribution of this code doesn't not have a license file, the spec file from which I've based a bunch of the metadata lists `license: ?`.

Reply:
this package only exists to service a number of downstream packages in the same series, which are all GPL, so if a license is _required_, I will release this under GPL.

Reply:
Please list it as GPL 2+ as that is what the license should have been.

Reply:
@emaros, done in 472e298.